src/forms/SparkForm/SparkForm.js

```diff
@@ -124,7 +122,7 @@ const SparkForm = ({
           </Link>
         )}
       </RadioInput>
-      <div>
+      <div className="tags">
```

Comment:
Classnames should be inherited from the `style` object imported in the components